lib/mapping/CMap.h

```diff
@@ -90,12 +89,9 @@ struct DLL_LINKAGE PlayerInfo
 	/// WARNING: powerPlaceholders sometimes gives false 0 (eg. even if there is one placeholder), maybe different meaning ???
 	ui8 powerPlaceholders;
 
-	template <typename Handler>
-	void serialize(Handler & h, const int version)
+	template<typename Handler> void serialize(Handler & h, const int version)
 	{
-		h & p7 & hasRandomHero & mainCustomHeroId & canHumanPlay & canComputerPlay & aiTactic & allowedFactions & isFactionRandom &
-				mainCustomHeroPortrait & mainCustomHeroName & heroesNames & hasMainTown & generateHeroAtMainTown &
-				posOfMainTown & team & generateHero;
+		h & p7 & hasRandomHero & mainCustomHeroId & canHumanPlay & canComputerPlay & aiTactic & allowedFactions & isFactionRandom & mainCustomHeroPortrait & mainCustomHeroName & heroesNames & hasMainTown & generateHeroAtMainTown & posOfMainTown & team & generateHero;

```

Comment:
Thanks for pointing out. I think for the sake of desync debugging we should really put all of serializer statements on own lines:
```
h & p7;
h & hasRandomHero;
h & mainCustomHeroId;
...
```
Probably should submit PR for that.